pom.xml

```diff
@@ -4,9 +4,9 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>com.amazonaws</groupId>
+  <groupId>com.webs</groupId>
   <artifactId>amazon-sqs-java-messaging-lib</artifactId>
-  <version>1.0.4</version>
+  <version>1.0.5</version>
   <packaging>jar</packaging>
```

Comment:
we need to add a publishing config that points to our nexus repository.  see https://maven.apache.org/pom.html#Repository

I think we add the following

```
  <distributionManagement>
    <repository>
      <uniqueVersion>false</uniqueVersion>
      <id>nexus-releases</id>
      <name>Nexus Releases Repository</name>
      <url>http://nexus.digital.vistaprint.io/repository/releases/</url>
    </repository>
    <snapshotRepository>
      <uniqueVersion>true</uniqueVersion>
      <id>nexus-snapshots</id>
      <name>Nexus Snapshots Repository</name>
      <url>http://nexus.digital.vistaprint.io/repository/snapshots/</url>
    </snapshotRepository>
  </distributionManagement>
```
somewhere on this top level of the pom.xml